src/lib/containers/synapse_table_functions/SynapseTableCell.tsx

```diff
@@ -262,22 +255,7 @@ export const SynapseTableCell: React.FC<SynapseTableCellProps> = ({
     case ColumnType.INTEGER:
     case ColumnType.BOOLEAN:
     case ColumnType.LARGETEXT: {
-      const isShort = isShortString(columnValue)
-      if (isShort) {
-        return <p className={isBold}> {columnValue}</p>
-      } else {
-        return (
-          <div className={isBold}>
```

Comment:
Dropped this div (which I'm guessing might have been used to style the tooltip?)